components/Post/PostContent.tsx

```diff
@@ -31,9 +37,14 @@ const PostContent = ({ post }: Props) => {
       </Head>
 
       <div className="blogpost-image">
+        {isPostADraft(post) ? <DraftPostMark /> : null}
+        {isPostInTheFuture(post) === false ? <FuturePostMark /> : null}
         <img
-          src={post.featuredImage ?? "/post-images/placeholder.png"}
+          src={
+            post.featuredImage ? post.featuredImage : "/post-images/draft.webp"
+          }
           alt="blog post image"
+          className="gray-filter-for-img"
```

Comment:
![image](https://user-images.githubusercontent.com/46556479/200105130-74f41501-3b35-4f45-acaa-f91e5eb66af8.png)
This pictures, has this class, but does not make any effect 🤯